_posts/2016-10-26-jwt-artsy-journey.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "JWT, Artsy Journey"
```

Comment:
_...Artsy's Journey_

or maybe even:

_JSON Web Tokens: Artsy's Journey_

for better keyword matching!